blockly-core/core/ui/contract_editor/example_view.js

```diff
@@ -52,14 +50,19 @@ Blockly.ExampleView.prototype.initializeTestButton = function (buttonText, iconC
 };
 
 Blockly.ExampleView.prototype.testExample_ = function () {
-  this.resultText.innerHTML = this.onExampleRun_(this.block_);
+  this.contractEditor_.resetExampleViews();
```

Comment:
That could work! For emitters in closure-land, see e.g.

https://github.com/code-dot-org/code-dot-org/blob/vertical_scrollbars/blockly-core/core/ui/block_space/block_space.js#L79

or

https://github.com/code-dot-org/code-dot-org/blob/vertical_scrollbars/blockly-core/core/ui/block.js#L131-131